_posts/blog/2015-04-24-data_duplication_writeup.md

```diff
@@ -26,6 +26,7 @@ This issue was introduced into the dataset sometime in May 2014. We fixed the ro
 
 The table below details the effect of the duplication bug on each of the BigQuery tables. Note that the table is organized not by time of query, but according to the time the M-Lab test occurred, as that is how M-Lab data is partitioned into tables in BigQuery.
 
+{:.big-query}
```

Comment:
I like making it an id instead of a class, you make a good point there.  I am hoping the base styles for tables should be more flexible moving forward, so I don't think we need anything special at the moment for large tables.  It probably just depends on how much bigger some future tables get though, so we may need to revisit in the future.